pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -25,8 +25,9 @@ type Retriever struct {
 }
 
 const (
-	bindingPrefix = "SERVICE_BINDING"
-	secretPrefix  = "urn:alm:descriptor:servicebindingrequest:env:object:secret"
+	bindingPrefix   = "SERVICE_BINDING"
+	secretPrefix    = "urn:alm:descriptor:servicebindingrequest:env:object:secret"
+	configMapPrefix = "urn:alm:descriptor:servicebindingrequest:env:object:configmap"
```

Comment:
I think we should add a `basePrefix` to hold `urn:alm:descriptor:servicebindingrequest:env:object`, which is always common, and then:

``` go
configMapPrefix = fmt.Sprintf("%s:secret", basePrefix)
```